python/python/analysis.c

```diff
@@ -169,7 +169,7 @@ static int py_analysis(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *bu
 		};
 		PyObject *memview = PyMemoryView_FromBuffer (&pybuf);
 		PyObject *arglist = Py_BuildValue ("(NK)", memview, addr);
-		PyObject *result = PyEval_CallObject (py_analysis_cb, arglist);
+		PyObject *result = PyObject_CallObject (py_analysis_cb, arglist);
```

Comment:
I suggest to put `.clang-format` here - the same as in the main Rizin repo and reformat the code as well.